packages/components/src/focal-point-picker/grid.js

```diff
@@ -0,0 +1,79 @@
+/**
+ * WordPress dependencies
+ */
+import { useRef, useEffect, useState } from '@wordpress/element';
+
+/**
+ * Internal dependencies
+ */
+import {
+	GridView,
+	GridLineX,
+	GridLineY,
+} from './styles/focal-point-picker-style';
+
+export default function FocalPointPickerGrid( {
+	bounds = {},
+	percentages = {
```

Comment:
Seems like the only thing we need here is a new instance of the value when it changed. Do you think we should just name this `value` and avoid any computation/destructuring? As written it gives the impression that we do need each value separately but by looking at the code it's not really the case, we just need a new instance.

Reply:
Works for me